system-addon/common/Actions.jsm

```diff
@@ -39,6 +39,7 @@ for (const type of [
   "NEW_TAB_INIT",
   "NEW_TAB_INITIAL_STATE",
   "NEW_TAB_LOAD",
+  "NEW_TAB_RENDERED",
```

Comment:
Can you rename this to `"NEW_TAB_REHYDRATED"` since that's closer to what it is?